packages/wallet-ui/src/utils/utils.ts

```diff
@@ -79,3 +80,23 @@ export const fetchWithTimeout = async (
   clearTimeout(id);
   return response;
 };
+
+export const getDate = (date: number, options?: any) => {
+  const config = options || { month: 'short', day: 'numeric' };
+  return new Date(date).toLocaleDateString('en-US', config);
```

Comment:
Not sure if it is available in the snap context, but you could use [navigator.language](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language) here and in other needed spots instead to localize to a user's preference. Low priority/ok to skip.

Reply:
fixed